docs/source/usage/workflows/tracerParticles.rst

```diff
@@ -8,21 +8,78 @@ Tracer Particles
 Tracer particles are like :ref:`probe particles <usage-workflows-probeParticles>`, but interact self-consistenly with the simulation.
 They are usually used to visualize *representative* particle trajectories of a larger distribution.
 
+In PIConGPU each species can be a tracer particle species, this allows tracking fields along the particle trajectory.
+
 Workflow
 """"""""
 
-* ``speciesDefinition.param``: create a species specifically for tracer particles
+* ``speciesDefinition.param``: add the particle attribute ``particleId`` to your species to give each particle a unique id
+
+  .. code-block:: cpp
+
+      using ParticleFlagsTracerElectrons = MakeSeq_t<
+          particlePusher< particles::pusher::Boris >,
+          shape< UsedParticleShape >,
+          interpolation< UsedField2Particle >,
+          currentSolver::EmZ<UsedParticleCurrentSolver>
+      >;
+
+      using TracerElectron = Particles<
+          PMACC_CSTRING( "e_tracer" ),
+          ParticleFlagsTracerElectrons,
+          MakeSeq_t<
+              position< position_pic >,
+              momentum,
+              weighting,
+              particleId
+          >
+      >;
+
+* ``speciesDefinition.param``: create a species specifically for tracer particles which only stores ``probeE``, the electric interpolated field
+
+  .. code-block:: cpp
+
+      using ParticleFlagsTracerElectrons = MakeSeq_t<
+          particlePusher< particles::pusher::Boris >,
+          shape< UsedParticleShape >,
+          interpolation< UsedField2Particle >,
+          currentSolver::EmZ<UsedParticleCurrentSolver>
+      >;
 
-  * add the particle attribute ``particleId`` to your species' ``Particles< ... >`` class (third argument, ``T_Attributes``)
-  * optional: add ``fieldE`` and ``fieldB`` attributes to the species to store fields as in :ref:`probes <usage-workflows-probeParticles>` 
+      using TracerElectron = Particles<
```

Comment:
Hmm, I wonder if it makes sense to have two code blocks for this.
Usually, when I have a tracer particle with E-field recording, I'd also like to have its ID to follow its path.
IMO one code block that adds both `particleId` and `probeE` is enough.

Reply:
> Usually, when I have a tracer particle with E-field recording, I'd also like to have its ID to follow its path

Maybe it's worth putting this explicitly before the code, and then express in the code?

Reply:
@n01r I kept the *ID* and *probe* separate. The id is used to follow a particle where the probe attribute is what the particle sees during the walk. I do not want the user thinking an *ID* is required for tracer species.